### PR TITLE
fix(BUX-466): handle p2p authch message

### PR DESCRIPTION
**Pull request by dorzepowski**
# Pull Request Checklist

- [x] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/pulse/blob/main/.github/CODE_STANDARDS.md)
- [x] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/pulse/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] 🏠 I tested my changes locally.
- [x] ✅ I have provided tests for my changes.
- [x] 📝 I have used conventional commits.
- [x] 📗 I have updated any related documentation.
- [x] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: internal/wire/message.go [hunk 1]
```diff
@@ -186,6 +187,9 @@ func makeEmptyMessage(command string) (Message, error) {
 	case CmdProtoconf:
 		msg = &MsgProtoconf{}
 
+	case CmdAuthch:
```
Comment (chris-4chain) [inline internal/wire/message.go:190]: Maybe
```golang
case CmdProtoconf, CmdAuthch:
    msg = &MsgProtoconf{}
```

Comment (kuba-4chain) [reply internal/wire/message.go:190]: But it should not be the MsgProtoconf message. It should be a MsgAuthch?

Comment (arkadiuszos4chain) [reply internal/wire/message.go:190]: Yes, it's too dirty fix, I beleive we should implement "empty" handler for authch command

Comment (arkadiuszos4chain) [reply internal/wire/message.go:190]: Since we probably will change entire code responsible for peer communication it can stay as it is

**Review by arkadiuszos4chain — APPROVED** [review]

**Pull request merged**